app/src/editor/shared/components/ModuleHeader/HamburgerButton/index.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import React, { useCallback } from 'react'
+import cx from 'classnames'
+import styles from './hamburgerButton.pcss'
+
+type Props = {
+    className?: string,
+    onFocus?: ?(e: SyntheticEvent<EventTarget>) => void,
+}
+
+const HamburgerButton = ({ className, onFocus: onFocusProp, ...props }: Props) => {
+    const onFocus = useCallback((e: SyntheticEvent<EventTarget>) => {
+        e.stopPropagation()
+
+        if (onFocusProp) {
+            onFocusProp(e)
+        }
+    }, [onFocusProp])
```

Comment:
IIUC the `stopPropagation` is here to prevent bubbling to the Module's focus handler? Perhaps this belongs in `Module.jsx` or `ModuleHeader.jsx` instead? Maybe